doc/website/blog/title-tbd.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "TBD"
+author: Jeremy Vesperman
+authorURL: "https://github.com/jeremyv2014"
+authorImageURL: "https://avatars.githubusercontent.com/u/9276415?v=4"
+---
+
+I have an embarrassing secret to share. I've been writing software for years, but, until recently, I 
```

Comment:
I don't think it's embarrassing at all. I rather we omit this phrase (up to you), but I think this sentence works without embarrassment of knowledge gaps.

Reply:
You can say, "you may find it surprising" , or something similar if you like.